### PR TITLE
fix:alignment of buttons in basic-details route

### LINKED ISSUE #490: Improper alignment of buttons(mobile view) in basic details route
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**
screen shot 2017-07-17 at 5 49 01 pm

**Expected behavior:**
screen shot 2017-07-17 at 5 48 29 pm

I would send a PR for this

### PR DESCRIPTION
**Pull request by utkarshnath**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

Fixes #490 
https://young-wildwood-91762.herokuapp.com/create

### CODE REVIEW

**Comment by sumedh123** [comment]
@utkarshnath The text on the buttons is overflowing to the icon part also.
create_buttons

File: app/templates/components/forms/wizard/basic-details-step.hbs [hunk 1]
```diff
@@ -531,7 +531,7 @@
     {{/ui-dropdown}}
   </div>
   <br>
-  <div class="{{unless device.isMobile 'large right floated'}} ui buttons">
+  <div class="{{unless device.isMobile 'right floated' 'three'}} ui buttons large">
```
Comment (CosmicCoder96) [inline app/templates/components/forms/wizard/basic-details-step.hbs]: @utkarshnath the `large` class was removed for mobile for the same reasons as pointed in the comment by @sumedh123 , please place it back there.

File: app/templates/components/forms/wizard/sessions-speakers-step.hbs [hunk 2]
```diff
@@ -322,7 +322,7 @@
     <div class="spacer-50"></div>
   {{/if}}
 
-  <div class="ui buttons {{if device.isMobile 'fluid' 'large right floated'}}">
+  <div class="ui large buttons {{if device.isMobile 'fluid' 'right floated'}}">
```
Comment (CosmicCoder96) [inline app/templates/components/forms/wizard/sessions-speakers-step.hbs]: same here.

File: app/templates/components/forms/wizard/sponsors-step.hbs [hunk 3]
```diff
@@ -72,7 +72,7 @@
     <div class="spacer-50"></div>
   {{/if}}
 
-  <div class="{{if device.isMobile 'mini three' 'large right floated'}} ui buttons">
+  <div class="{{if device.isMobile 'three' 'right floated'}} ui buttons large">
```
Comment (CosmicCoder96) [inline app/templates/components/forms/wizard/sponsors-step.hbs]: same here.

**Comment by geekyd** [comment]
@utkarshnath Please also change the `large` class in the parent div for mobile device. That will fix the misaligned text of the buttons mentioned above.

File: app/templates/components/forms/wizard/basic-details-step.hbs (hunk 1 above)
Comment (utkarshnath) [inline app/templates/components/forms/wizard/basic-details-step.hbs]: done

**Comment by utkarshnath** [comment]
@geekyd @CosmicCoder96 @sumedh123 plz review

**Comment by anu-007** [comment]
@utkarshnath there is still an overflow on smaller screen
image

**Comment by geekyd** [comment]
@anu-007 What is the resolution of the screen?

**Comment by anu-007** [comment]
@geekyd (320x568) iPhone5 in chrome dev tools.

**Comment by geekyd** [comment]
@anu-007 It looks fine to me, please check again may be the deployment was not updated.

**Comment by anu-007** [comment]
@geekyd I checked it in incognito mode

**Comment by geekyd** [comment]
@anu-007 My bad, the problem exists.
@utkarshnath Please add `mini` class to the parent div if the device is mobile.

**Comment by utkarshnath** [comment]
@anu-007 thanks for pointing that out, updated. Plz review

**Comment by geekyd** [comment]
@utkarshnath Please update the deployment link. It is not working.

**Comment by utkarshnath** [comment]
@geekyd updated

**Pull request merged**